code-reviews/pull-requests.md

```diff
@@ -33,17 +33,31 @@ We should always aim to have pull requests be as small as possible, without losi
 1. They are easier to deploy; this is aligned with the strategy of release fast and release often.
 1. Minimizes possible conflicts and stale PRs, which are difficult to merge and keep in sync with master either because they're very dynamic or contain refactoring.
 
-However, we should avoid having PRs that include code that is without context or loosely coupled.
+However, we should avoid having PRs that include code that is without context or loosely coupled. There is no right size, but keep in mind that a code review is a collaborative process, and you should work with your reviewer to find out a reasonable size. Besides the size, remember that every PR should:
 
-<!-- TODO: Expand the topics below
+1. Be consistent and within context, 
+1. not break the build, and
+1. include related tests (UTs, integration, etc.) as part of the PR.
+ 
+There are times where seems a big PR is unavoidable and necessary; the act of breaking it up would be more time consuming and error-prone than it would be beneficial, and that it is the reason why it's easier to write small PRs in the first place.
```

Comment:
This sentence doesn't feel necessary, the gist of writing small PRs is still there, I would suggest removing it.

Reply:
This is based on previous conversations/discussions/PR reviews where other developers point is that sometimes PR seem unavoidable and necessary, I thought it would clarify that when that happens it is true that it is more time consuming breaking the PR, but we should aim to write PRs small in the first place. I still think is valuable to make it clear, but if I can take it that maybe is too much.

Reply:
Then I would suggest a rephrase to:

Start small, it is easier to create a small PR from the start than to break up a bigger one.

Reply:
That is an amazing way to summarize it, rephrased.